discount/src/main/java/com/kyriba/discount/web/controller/DiscountController.java

```diff
@@ -0,0 +1,60 @@
+package com.kyriba.discount.web.controller;
+
+import com.kyriba.discount.domain.DiscountType;
+import com.kyriba.discount.domain.dto.DiscountDto;
```

Comment:
domain objects in API

Reply:
I don't see any reason to add more objects right now. If I need an object to manipulate domain logic, then I will add this one in the future.